### DISCUSSION TITLE
Run a server in parallel and pass variables from server to client

### DISCUSSION THREAD

**Discussion by DrakoPOD** [quasarframework/quasar#10458] [CLI - SSR mode]
I trying to make a web app for visualize IoT data (sensor and actuators), I need to run a MQTT Broker in parallel, also I need to access to broker data direct from server, due by the limitations about MQTT in client side.

How I configure Node Index.js file to add the broker?
How to access to a server variable from client?

Edit:
`useSSRContext` doesn't work a the same way that in vanilla Vue, Quasar already use it to pass server-client parameters. So I need other way to pass my own variables.

**Comment by ontwikkelfabriek** [comment]
@DrakoPOD 

Quasar is client side only, but there's a new feature called `SSR Middleware`:
https://next.quasar.dev/quasar-cli/developing-ssr/ssr-middleware#anatomy-of-a-middleware-file

**Reply by DrakoPOD** [reply]
Thanks, now I can run a new server in parallel, well, kind of, because the server never close when the page reload for changes and the port is still in use and I get an error. But I don't understand `ssrContext`, doesn't work at the same way that on vanilla vue, so I can't pass variables from server to client.
